importer/src/main.rs

```diff
@@ -213,3 +224,27 @@ fn oneshot(osm_path: String, clip: Option<String>, drive_on_right: bool, build_c
     timer.stop("save map");
     println!("{} has been created", abstutil::path_map(&name));
 }
+
+fn are_dependencies_callable(job: &Job, config: &ImporterConfiguration) -> bool {
```

Comment:
Could we move this to `confguration.rs`, maybe putting the `job.osm_to_raw` check at the call-site, so something like `if job.raw_to_osm { if !config.are_dependencies_callable() { print and exit } }`

Reply:
I thought the dependencies might depend on the job in a more complicated way, but it turned out not to be the case, so I can do this. I don't think it quite fits with configuration.rs, but I can make another module file for it.